shared/src/types/employers.ts

```diff
@@ -0,0 +1,10 @@
+import { DocumentReference, Timestamp } from 'firebase-admin/firestore';
+
+export const EMPLOYERS_FIRESTORE_PATH = 'employers';
+
+export type Employer = {
+	userId: DocumentReference;
```

Comment:
A little nitpicky, but can we use underscores in the name here, i.e. `user_id`, `is_current`, etc? The document fields in firestore are usually with underscores and I would like to be consistent there if possible🙈